### ISSUE TITLE
kubeadm pre-flight output message needs fixing

### ISSUE THREAD

**Issue by mikedanese** [kubernetes/kubeadm#38]
_From @errordeveloper on October 17, 2016 7:39_

```
<cmd/init> Running pre-flight checks
<preflight/checks> WARNING: kubelet service is not enabled, please run 'systemctl enable kubelet.service'

kubeadm: I am an alpha version, my authors welcome your feedback and bug reports
kubeadm: please create issue an using https://github.com/kubernetes/kubernetes/issues/new
kubeadm: and make sure to mention @kubernetes/sig-cluster-lifecycle. Thank you!
<cmd/init> preflight check error
  count: 2 
  msg: 
    ebtables not found in system path
    ethtool not found in system path
```

Firstly, it has some typos. Secondly, it doesn't tell the user how to skip the checks.

I'd like it to look more like this:

```
<cmd/init> running pre-flight checks
<preflight/checks> WARNING: kubelet service is not enabled, please run 'systemctl enable kubelet.service'

kubeadm: I am an alpha version, my authors welcome your feedback and bug reports
kubeadm: please create an issue using https://github.com/kubernetes/kubernetes/issues/new
kubeadm: and make sure to mention @kubernetes/sig-cluster-lifecycle. Thank you!
<cmd/init> preflight check error - 2 check(s) failed:
    - ebtables not found in system path
    - ethtool not found in system path
<cmd/init> if you know what you are doing, you can skip pre-flight checks with `--skip-preflight`
```

_Copied from original issue: kubernetes/kubernetes#34927_

**Comment by mikedanese** [comment]
_From @pires on October 17, 2016 8:25_

This output is not like this anymore. Did you run the latest in master?

**Comment by mikedanese** [comment]
_From @errordeveloper on October 17, 2016 10:16_

@pires, thanks, will check latest.

**Comment by mikedanese** [comment]
_From @errordeveloper on October 17, 2016 11:16_

Ok, so new output format looks like this:
```
Running pre-flight checks
WARNING: no supported init system detected, skipping checking for services
WARNING: no supported init system detected, skipping checking for services
WARNING: tc not found in system path
preflight check errors:
	user is not running as root
	ebtables not found in system path
	ethtool not found in system path
	ip not found in system path
	iptables not found in system path
	nsenter not found in system path
	socat not found in system path
```

I still think we should add suggestion about the flag.
```
<cmd/init> if you know what you are doing, you can skip pre-flight checks with `--skip-preflight`
```

Also, I think it's too early to remove alpha status message. If people don't like how it was phrase, let's review, but it needs to be there.

**Comment by mikedanese** [comment]
_From @pires on October 17, 2016 13:15_

As per pre-flight checks PR, alpha warning only shows up on errors other than pre-flight ones.

**Comment by luxas** [comment]
PR: https://github.com/kubernetes/kubernetes/pull/37568

**Comment by luxas** [comment]
Fixed with kubernetes/kubernetes#37568
